pkg/transport/process/util.go

```diff
@@ -0,0 +1,177 @@
+package process
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"time"
+
+	cdv2 "github.com/gardener/component-spec/bindings-go/apis/v2"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	componentDescriptorFile = "component-descriptor.yaml"
+	resourceFile            = "resource.yaml"
+	resourceBlobFile        = "resource-blob"
+)
+
+// WriteTARArchive writes the component descriptor, resource and resource blob to a TAR archive
```

Comment:
can you also add to the description that this is the communication format that is used in the processors.
Maybe we should also name accordingly. So that its not just a tarring function but a encode process message function